pkg/entities/webhook.go

```diff
@@ -464,6 +496,15 @@ func (e *Entity) WebhookUpvoteStreak(userID, source string) error {
 		return fmt.Errorf("failed to update user upvote streak: %v", err)
 	}
 
+	// handle quest logs
+	log := &model.QuestUserLog{
+		UserID: userID,
+		Action: model.QuestAction(model.VOTE),
+	}
+	if err := e.UpdateUserQuestProgress(log); err != nil {
+		e.log.Fields(logger.Fields{"log": log}).Error(err, "[entity.WebhookUpvoteStreak] entity.UpdateUserQuestProgress() failed")
+	}
+
```

Comment:
should we return here

Reply:
return err here may prevent below functions from execution
this quest handler should not affect original function flow imo